provider/tuningconfigs/resource.go

```diff
@@ -369,10 +384,10 @@ func getTuningConfigBuilder(ctx context.Context, plan *TuningConfig) (*cmv1.Tuni
 }
 
 func parseInputString(input []byte) (map[string]interface{}, error) {
-	var validSpecJson map[string]interface{}
-	err := json.Unmarshal(input, &validSpecJson)
+	var validSpec map[string]interface{}
+	err := yaml.Unmarshal(input, &validSpec)
```

Comment:
Does this library support to unmarshal both json and yaml input using the same Unmarshal function?

Reply:
Yes, https://github.com/openshift/rosa/blob/f012d59c3b072b9ffa6cdcd8ddd9d5647855b007/pkg/input/helper.go#L25 it is in use for the cli currently